src/libostree/ostree-kernel-args.c

```diff
@@ -202,14 +531,24 @@ _ostree_kernel_args_append_proc_cmdline (OstreeKernelArgs *kargs,
   g_strchomp (proc_cmdline);
 
   proc_cmdline_args = g_strsplit (proc_cmdline, " ", -1);
-  _ostree_kernel_args_append_argv_filtered (kargs, proc_cmdline_args,
+  ostree_kernel_args_append_argv_filtered (kargs, proc_cmdline_args,
                                             filtered_prefixes);
 
   return TRUE;
 }
 
+/*
+ * Note: this is a new function added to the API
```

Comment:
I think we could remove these `new function added to the API` lines now - think they were there before to indicate a new function added to `rpm-ostree` but not yet in the `ostree` API?

Reply:
I see.. Thank you for explaining!